src/sentry/integrations/slack/utils.py

```diff
@@ -295,7 +295,13 @@ def build_group_attachment(group, event=None, tags=None, identity=None, actions=
     }
 
 
-def build_incident_attachment(incident):
+def build_incident_attachment(incident, metric_value=None):
+    """
+    Builds an incident attachment for slack unfurling
+    :param incident: The `Incident` to build the attachment for
+    :param metric_value: The value
```

Comment:
Lol this is really helpful.

Reply:
Lol, I must have gotten distracted part way through writing that